themes/src/main/resources/theme/keycloak.v2/login/field.ftl

```diff
@@ -65,7 +65,7 @@
         </span>
       </div>
       <div class="${properties.kcInputGroupItemClass}">
-        <button class="pf-v5-c-button pf-m-control" type="button" aria-label="${msg('showPassword')}"
+        <button tabindex="-1" class="pf-v5-c-button pf-m-control" type="button" aria-label="${msg('showPassword')}"
```

Comment:
Why would you not want this element to be focussable with keyboard navigation?

Reply:
IMO it's not nice to focus the password reveal button

Reply:
It still needs to be accessible users with an impairment no? E.g. from [WebAIM](https://webaim.org/techniques/keyboard/tabindex#zero-negative-one):

> Remember that `tabindex="-1"` removes the element from the default navigation flow. Do not assign `tabindex="-1"` to any element that must be keyboard navigable, such as a link or button that sighted users can click on with the mouse.

Reply:
This button reveals the password that you typed on the screen, so that is only something that sighted people would care about no?

Reply:
My perspective on this: There are people who have difficulties to use a mouse, for example to position the cursor at the right position and then pressing the button of the mouse at the right moment without moving the mouse again.  Imagine an elderly person which has acquired a condition, or a worker which that has injured their hand. 

Those are better off using the keyboard, than the mouse, and they are find in seeing what is on the screen. As they might have problems pressing the right keys on their keyboard, they are candidates to really want to show their passwords to check if they typed them correctly before sending them. 

So I'd be +1 for having a tabindex for that field.

Reply:
Yeah, I think we should really keep this accessible